_specs/ecip-_.md

```diff
@@ -0,0 +1,103 @@
+---
+ecip: TBD
```

Comment:
You can use number 1077.

Reply:
Done via https://github.com/ethereumclassic/ECIPs/pull/224/commits/6e9b361c2af084af9fcee9089cbad538b21e41d8